README.md

```diff
@@ -72,11 +72,22 @@ end
 
 To install the latest release as a gem:
 
-`gem install rack-test`
+```
+gem install rack-test
+```
 
 Or via Bundler:
 
-`gem "rack-test", require: "rack/test"`
+```
+gem 'rack-test', require: 'rack/test'
```

Comment:
I changed "rack/test" to 'rack/test'. This modification is because `rubocop` is valid for the single quote code, and it looks more popular.